packages/core/src/index.ts

```diff
@@ -17,7 +17,8 @@
 export * from './api';
 export { default as Page } from './layout/Page';
 export { gradients, pageTheme } from './layout/Page';
-export type { PageTheme } from './layout/Page';
+import type { PageTheme } from './layout/Page';
+export type { PageTheme };
```

Comment:
We can revert all of these once https://github.com/alangpierce/sucrase/pull/533 is released

Reply:
... which it has been now, so changes reverted :grin: